Sources/App/WebView/IncomingURLHandler.swift

```diff
@@ -5,7 +5,7 @@ import SafariServices
 import Shared
 
 class IncomingURLHandler {
-    let windowController: WebViewWindowController
+    weak var windowController: WebViewWindowController?
```

Comment:
```suggestion
    private(set) weak var windowController: WebViewWindowController?
```

Reply:
Given some of the cases below crash, might as well just make this a `!` instead of `?`